synthtool/gcp/gapic_bazel.py

```diff
@@ -200,7 +200,7 @@ def _clone_googleapis(self):
 
         else:
             log.debug("Cloning googleapis.")
-            self._googleapis = git.clone(GOOGLEAPIS_URL, depth=1)
+            self._googleapis = git.clone(GOOGLEAPIS_URL)
```

Comment:
Note that the entire git history may be huge. Maybe it doesn't matter that much as the cache directory is shared across the entire synth run.

Reply:
In the case where the git history is huge, what is the negative effect?  Will generation fail?

Reply:
Also, notice in git.py, I only download the single branch master, so it won't download the entire history of the repo.

Reply:
It's probably premature optimization and won't really matter in practice. If it is an issue, we can revisit.